wolfcrypt/src/aes.c

```diff
@@ -3425,7 +3425,8 @@ static void GHASH(Aes* aes, const byte* a, word32 aSz,
     GMULT(x, h);
 
     /* Copy the result into s. */
-    XMEMCPY(s, x, sSz);
+    (sSz > AES_BLOCK_SIZE) ? XMEMCPY(s, x, AES_BLOCK_SIZE)
+                           : XMEMCPY(s, x, sSz);
 }
```

Comment:
You might consider something more consistent with our coding style like:

if (sSz > AES_BLOCK_SIZE) sSz = AES_BLOCK_SIZE;
XMEMCPY(s, x, sSz);

Its more common to use the "exp ? true : false" syntax when setting a variable or returning a value.

Same applies for others below.